lightkube/core/async_client.py

```diff
@@ -416,7 +417,7 @@ async def stream_log():
             resp = await self._client.send(req, stream=follow)
             self._client.raise_for_status(resp)
             async for line in resp.aiter_lines():
-                yield line
+                yield transform_newlines(newlines)(line)
```

Comment:
Since we are now targeting httpx >= 0.24, can we just assume the new line is not returned?
Doing so the code would be simpler / faster:

```suggestion
                yield line + '\n' if newlines else line
```

Reply:
sure, this is fine so long as no one has httpx pinned back for some reason.  I don't :)